crates/starknet-devnet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -293,7 +286,12 @@ impl JsonRpcHandler {
             Err(e @ Error::NoStateAtBlock { .. }) => {
                 Err(ApiError::NoStateAtBlock { msg: e.to_string() })
             }
-            Err(err) => Err(ApiError::ContractError { error: err }),
+            Err(Error::ContractExecutionError(error_stack)) => {
+                Err(ApiError::ContractError { error_stack })
+            }
+            Err(e) => Err(ApiError::ContractError {
+                error_stack: ErrorStack::from_str_err(&e.to_string()),
+            }),
```

Comment:
Here and in three other places in this file, our wildcard error used to be `ContractError`. I kept the same wildcard error, but perhaps that is misuse of that error, and `ApiError::StarknetDevnetError` could be used.